.github/workflows/python-package.yml

```diff
@@ -60,7 +60,15 @@ jobs:
       run: |
         sudo apt update
         sudo apt -y install maven samtools bcftools parallel libbz2-dev liblzma-dev
+    - name: Set up ascp binary
+    # Might need to periodically update this URL
+      run: |
+        curl https://d3gcli72yxqn2z.cloudfront.net/connect_latest/v4/bin/ibm-aspera-connect_4.1.3.93_linux.tar.gz -o ibm-aspera-connect.tar.gz
+        tar xzvf ibm-aspera-connect.tar.gz
+        ./ibm-aspera-connect_*_linux.sh
+        sudo ln -s ~/.aspera/connect/bin/ascp /usr/bin
```

Comment:
Some distros might not allow for binaries to run off known binary folders unless the execute permission is set. So might as well do a "chmod +x" here...

Reply:
`ascp` always come already executable from the downloaded package. It does not hurt to add the `chmod +x`

Reply:
done